cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -66,13 +66,23 @@ set(configure_command
 # Pass C flags through
 set(configure_opts ${configure_opts} ${CMAKE_C_FLAGS})
 
+# Pass BUILD_SHARED_LIBS (what about RPATH?)
+string(COMPARE EQUAL "@BUILD_SHARED_LIBS@" "ON" is_shared_lib)
```

Comment:
Can't use this because it may be `YES` or `1`. Just use `if(BUILD_SHARED_LIBS)`.